src/client/datascience/raw-kernel/rawJupyterSession.ts

```diff
@@ -262,19 +248,28 @@ export class RawJupyterSession extends BaseJupyterSession {
         );
 
         this.terminatingStatus = undefined;
-        const process = await this.kernelLauncher.launch(
-            this.kernelConnectionMetadata,
-            this.launchTimeout,
+        const process = await KernelProgressReporter.wrapAndReportProgress(
             this.resource,
-            this.workingDirectory,
-            options.ui,
-            options.token
+            localize.DataScience.connectingToKernel().format(
```

Comment:
Why do we need `connectingToKernel`?
I thought we already displayed a message when we're connecting to a kernel.
THis just seems yet another but a duplicate.

Reply:
We didn't show this in some cases. It's just blank. And if this is redundant, it should have no effect since these things stack.